src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const reversedGender = (sex === 'm') ? 'wife' : 'husband';
```

Comment:
This name is a bit confusing 😄 You can just name it partner.

Reply:
@CharAnagn is right here 👍 
Also, ternaries should be multilined

Reply:
did not even think about it while typing but ye it sounds funny now 😄